svo (namedb).xml

```diff
@@ -1892,7 +1898,6 @@ temp_name_list[1].mark = "Ivory"</script>
 						<regexCodePropertyList>
 							<integer>1</integer>
 							<integer>1</integer>
-							<integer>-1509852592</integer>
```

Comment:
@vadi2 or @keneanung can you explain those integers? I have no idea what they are and why they are getting modified here, I only literally changed code lines and not messed with any triggers settings...

Reply:
It looks like an error to me. Is it always changing when you edit it?

Reply:
Looks like it... every time I rollback to the previous version and then edit and save the trigger it is doing that. Maybe it was some leftover from older mudlet versions?

Reply:
this is on 4.16?

could you show a screenshot of what pattern type are you setting that to?

Reply:
![image](https://user-images.githubusercontent.com/36555422/223709236-6462e305-ddd7-4700-abdc-66c15d266318.png)
This is how it looks like (I didn't change any of them, only the code)